wpeverest/everest-forms
# Enhancement - reCAPTCHA feature enhanced with reCAPTCHA language.

### All Submissions:



### Changes proposed in this Pull Request:
reCAPTCHA feature enhanced with reCAPTCHA language.

### How to test the changes in this Pull Request:

1. Go to Settings > reCAPTCHA and choose language from 'Choose reCAPTCHA Language' select box. And save the settings.
2. Create a form. 
3. Enable reCAPTCHA version that you have selected in  the global reCAPTCHA settings.
4. and Preview this form.
5. Verify, reCAPTCHA language changed or not , in language that you have selected.

### Types of changes:



### Other information:



### Changelog entry

> Enhancement - reCAPTCHA feature enhanced with reCAPTCHA language.
